lib/public/Notification/IAction.php

```diff
@@ -68,8 +69,9 @@ public function getLabel(): string;
 	/**
 	 * @param string $label
 	 * @return $this
-	 * @throws \InvalidArgumentException if the label is invalid
+	 * @throws InvalidValueException if the label is invalid
 	 * @since 9.0.0
+	 * @since 30.0.0 throws {@see \OCP\Notification\InvalidValueException} instead of \InvalidArgumentException
```

Comment:
```suggestion
	 * @since 30.0.0 throws {@see InvalidValueException} instead of \InvalidArgumentException
```
Or does the `@see` need fqdn?

Reply:
No, just had it in a wrong folder before and using the refactor in phpstorm didn't remove the fqdn